Description: Stop recording build date
 Recording the build date and time prevents gdbm to be build
 reproducibly. Given it was only added to the version string when
 certain macros were available,we simply disable the code
 unconditionally.
Author: Jérémy Bobbio <lunar@debian.org>
Reviewed-by: Dmitry Bogatov <KAction@gnu.org>
Last-Update: 2016-06-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/version.c
+++ b/src/version.c
@@ -25,9 +25,6 @@
    making the distdir. */
 const char * gdbm_version = "GDBM version " PACKAGE_VERSION ". "
 "03/01/2018"
-#if defined(__STDC__) && defined(__DATE__) && defined(__TIME__)
-		" (built " __DATE__ " " __TIME__ ")"
-#endif
 ;
 int const gdbm_version_number[3] = {
   GDBM_VERSION_MAJOR,
